build-meson-windows.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -eu
+
+PROJ_DIR=$PWD
+BUILD_DIR=$PROJ_DIR/build/meson
+DIST_DIR=$PROJ_DIR/windows-installer/win64-meson
+
+prepare()
+{
+    if [ -d ${PROJ_DIR}/plugins/windows-notification/yolort ]; then
+	echo "No need to re-download packages & yolort sources."
+    else
+        bash build-win64.sh --prepare
+    fi
+}
+
+configure()
+{
+    arg=${1:-"none"}
+    encr=${2:-"auto"}
+    local cmd=""
+    if [ x"${arg}" == x"reconfig" ]; then
+	    cmd=--reconfigure
+    fi
+    mkdir -p $BUILD_DIR
+    meson setup ${cmd} --prefix "$DIST_DIR" \
+	    -D crypto-backend=${encr} \
+	    -D plugin-ice=enabled \
+	    $PROJ_DIR $BUILD_DIR
+}
+
+build()
+{
+    cd $BUILD_DIR && ninja
+    ninja install
```

Comment:
No `cd` back to project root

Reply:
I believe  it is not needed as the each function will do cd $BUILD_DIR itself

Reply:
fixed.